code/game/objects/structures/som_cas_parts.dm

```diff
@@ -0,0 +1,52 @@
+/obj/structure/somcas
+	name = "\improper SOM Manta Jet"
+	desc = "The premier CAS VTOL of the SOM.Outmatches the TGMC Condor in speed and firepower with it's twin S-15 Fusion Engine it can reach an AO and lay heavy fire support within a short length of time."
```

Comment:
```suggestion
	desc = "The premier CAS VTOL of the SOM. Outmatches the TGMC Condor in speed and firepower with it's twin S-15 fusion engines, albeit very costly to produce. It can reach an AO and lay heavy fire support within a short length of time."
```